index.js

```diff
@@ -341,6 +352,17 @@
 //. // ...
 //. ```
 //.
+//. Individual Types can be included through jsdelivr:
+//. ```html
+//. <script src="https://cdn.jsdelivr.net/npm/sanctuary-maybe@2.1.0/index.js"></script>
+//. ```
+//.
+//. And accessed and aliased like so:
+//. ```javascript
+//. const Maybe = window.sanctuaryMaybe;
+//. // ...
+//. ```
+//.
```

Comment:
Might this be more at home in the [sanctuary-maybe][1] readme?


[1]: https://github.com/sanctuary-js/sanctuary-maybe

Reply:
From a complete beginner point of view, I would like to keep at least one example of using an individual one. Especially since this is the only website for Sanctuary.
Personally when im new to a framework I like to see the quick start option, but also seeing that using individual modules for production is an option. Also it illustrates that you can use the types without the framework.

I think this would benefit adoption and beginners.